### PR TITLE
Add release notes for WP 6.1

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

Add release notes for WP 6.1 outlining the changes noted in [this PR](https://github.com/pantheon-systems/WordPress/pull/344).

### Dependencies and Timing

**Release**:
- [x] After [PR #344](https://github.com/pantheon-systems/WordPress/pull/344) has been updated to point to these release notes, and has been approved.

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by jspellman814** [comment]
[Preview](https://pr-7871--pantheon-docs.my.pantheonfrontend.website/docs/start-states/wordpress)

**Review by jazzsequence — APPROVED** [review]
~LGTM~ ...actually 🤔 ...

**Review by CdrMarks — COMMENTED** [review]
@jspellman814 will we be adding an additional commit with a linked heading for the 6.1 release?

**Comment by jspellman814** [comment]
> @jspellman814 will we be adding an additional commit with a linked heading for the 6.1 release?

That's the plan! I was going to add a note to do this on [#344](https://github.com/pantheon-systems/WordPress/pull/344)

File: source/content/start-states/wordpress.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-s
 
 ## Latest Release
 
+### 2022-11-01
+
+#### <a name="20221018" class="release-update"></a>Replace mu-plugin with a format consistent with pantheon-systems/pantheon-mu-plugin
+
+This commit aligns the mu-plugin format to [our standalone repository](https://github.com/pantheon-systems/pantheon-mu-plugin), and will allow for the mu-plugin to receive updates from that repo whenever an updated version of WordPress is released. If you'd like to suggest changes to our mu-plugin, create an issue or open a PR [here](https://github.com/pantheon-systems/pantheon-mu-plugin).
```
Comment (jazzsequence) [inline source/content/start-states/wordpress.md:21]: using "here" as a link anchor is generally considered bad practice. Instead:

```suggestion
This commit aligns the mu-plugin format to [our standalone repository](https://github.com/pantheon-systems/pantheon-mu-plugin), and will allow for the mu-plugin to receive updates from that repo whenever an updated version of WordPress is released. If you'd like to suggest changes to our mu-plugin, create an issue or open a PR [in `pantheon-mu-plugin` issues](https://github.com/pantheon-systems/pantheon-mu-plugin/issues). 
```

**Review by jazzsequence — CHANGES_REQUESTED** [review]
Minor tweak to not use "here" as anchor text for a link

**Review by jazzsequence — APPROVED** [review]

**Comment by jazzsequence** [comment]
🚢

**Comment by whitneymeredith** [comment]
LGTM

**Pull request merged**